site/content/ytt/docs/develop/how-to-export-schema.md

```diff
@@ -28,6 +30,7 @@ load_balancer:
 #@schema/desc "DNS domains to accept traffic for."
 #@schema/default ["apps.example.com", "mobile.example.com"]
 app_domains:
+  #@schema/deprecated ""
```

Comment:
I think that it may be unlikely to have a schema with several deprecated data values, so to keep our example short, we should remove this `@schema/deprecated` and the two above it. Still keeping the two below.

Reply:
I agree, we want this example to be realistic, it would be unlikely for most of these keys to be deprecated independently. The one that seems most realistic is annotating `additional_config: {}`. Would you say we should just keep that one?

Reply:
Thanks for reviewing this PR. I have made the appropriate changes as suggested.